test/controllers/auth/FakeAuthAction.scala

```diff
@@ -16,9 +16,14 @@
 
 package controllers.auth
 
-import uk.gov.hmrc.play.frontend.auth.GovernmentGateway
+import play.api.mvc.{Request, Result}
 
-object GmpGovernmentGateway extends GovernmentGateway {
-  override val loginURL = ExternalUrls.signIn
-  override val continueURL = ExternalUrls.continue
+import scala.concurrent.Future
+
+object FakeAuthAction extends AuthAction {
+
+  override def invokeBlock[A](request: Request[A], block: AuthenticatedRequest[A] => Future[Result]): Future[Result] = {
+
+    block(AuthenticatedRequest("gmp/B1234567", request))
```

Comment:
Needs to be generated as it is an open service

Reply:
This isn't a nino, it's a psa id, does it still need to be generated?

Reply:
Actually it serves no purpose in the tests, it just needs to be there, so i have just changed it to "testID".

Reply:
Yeah that's fine. If it is any form of identifier it needs to be generated, just so people googling their identifiers are not confused